tests/msc3030_test.go

```diff
@@ -118,7 +118,7 @@ func TestJumpToDateEndpoint(t *testing.T) {
 			// Make the `/timestamp_to_event` request from Bob's perspective (non room member)
```

Comment:
Now that MSC3030 is merged and stable, do we also need to remove the `+build msc3030` build tags and change this to Synapse only instead?

Reply:
That's what I've been doing. 🤷

Reply:
👍 Copied from https://github.com/matrix-org/complement/pull/225/files

Added the Dendrite blacklist tags:
https://github.com/matrix-org/complement/blob/9b2ba1a3f8286b8f7e215b95ce8a5e5315c38d42/tests/room_timestamp_to_event_test.go#L1-L2